### PR TITLE
Subtask/as 23 lpa questionnaire setup environment

### PR DESCRIPTION
**Pull request by davehlewis**
## Ticket Number

AS-23

## Description of change
Created copy of forms-web-app and removed anything not required for lpa-questionnaire application. Simplified sub folders for controllers etc. accordingly.

## Checklist

- [X] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [X] My commit history in this PR is linear
- [X] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,173 +1,202 @@
...
-      - 4002:10000
-      - 4003:10001
-    volumes:
-      - ./tmp/blob-storage:/data
-
-  mongodb:
-    image: mongo:3.6.0
-    ports:
-      - 4000:27017
+# Port ranges (by convention):
```
Comment (MrSimonEmms) [inline docker-compose.yml:1]: I'm confused why this is showing the whole file in red/green - I'll figure it out, but just work out why

File: lpa-submissions-e2e-tests/cypress/integration/lpa-submission-tasklist/lpa-submission-tasklist.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps';
+
+Given('the lpa tasks are presented',()=>{
+    cy.visitTaskListQuestionnaireSubmissionPage();
+    
+});
+Given('And the task {string} is available for selection',()=>{
+
+});
```
Comment (MrSimonEmms) [inline lpa-submissions-e2e-tests/cypress/integration/lpa-submission-tasklist/lpa-submission-tasklist.js:9]: Add newline at end - use [editorconfig](https://editorconfig.org/) to do this for you

File: lpa-submissions-e2e-tests/cypress/plugins/index.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+module.exports = (on, config) => {
+  on('task', {
+    log (message) {
+      console.log(message)
+
+      return null
+    }
+  })
+}
```
Comment (MrSimonEmms) [inline lpa-submissions-e2e-tests/cypress/plugins/index.js:38]: Add newline at end - use [editorconfig](https://editorconfig.org/) to do this for you

File: lpa-submissions-e2e-tests/cypress/support/commands.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+            })
+        })
+    });
+}
+
+Cypress.Commands.add("checkPageA11y", (path) =>{
+    cy.visit(path);
+    cy.injectAxe();
+    cy.checkA11y(null,null,callback);
+})
```
Comment (MrSimonEmms) [inline lpa-submissions-e2e-tests/cypress/support/commands.js:34]: Add newline at end - use [editorconfig](https://editorconfig.org/) to do this for you

File: lpa-submissions-e2e-tests/cypress/support/lpa-submission-tasklist/visitTaskListQuestionnaireSubmissionPage.js [hunk 5]
```diff
@@ -0,0 +1,4 @@
+module.exports = () =>{
+    cy.visit('/lpa-submission/lpa-task-list')
+    cy.url().should('contain','/lpa-submission/lpa-task-list')
+}
```
Comment (MrSimonEmms) [inline lpa-submissions-e2e-tests/cypress/support/lpa-submission-tasklist/visitTaskListQuestionnaireSubmissionPage.js:4]: Add newline at end - use [editorconfig](https://editorconfig.org/) to do this for you

File: lpa-submissions-e2e-tests/package.json [hunk 6]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "lpa-submissions-e2e-tests",
+  "version": "1.0.0",
```
Comment (MrSimonEmms) [inline lpa-submissions-e2e-tests/package.json:3]: Set to `0.0.0` (and same in `package-lock.json`)

File: lpa-submissions-e2e-tests/package-lock.json [hunk 7]
```diff
@@ -0,0 +1,7246 @@
+{
+  "name": "lpa-submissions-e2e-tests",
+  "version": "1.0.0",
```
Comment (MrSimonEmms) [inline lpa-submissions-e2e-tests/package-lock.json:3]: Set to `0.0.0`

File: package-lock.json [hunk 8]
```diff
@@ -1652,6 +1652,12 @@
       "integrity": "sha512-v1plID3y9r/lPhviJ1wrXpLeyUIGAZ2SHNYTEapm7/8A9nLPoyvVp3RK/EPFqn5kEznyWgYZNsRtYYIWbuG8KA==",
       "dev": true
     },
+    "cypress-axe": {
```
Comment (MrSimonEmms) [inline package-lock.json:1655]: This appears to be in the wrong package - I'm not against this here if it's correct, but just check it

File: packages/lpa-questionnaire-web-app/patches/connect-mongodb-session+2.4.1.patch [hunk 9]
```diff
@@ -0,0 +1,48 @@
+diff --git a/node_modules/connect-mongodb-session/index.js b/node_modules/connect-mongodb-session/index.js
```
Comment (MrSimonEmms) [inline packages/lpa-questionnaire-web-app/patches/connect-mongodb-session+2.4.1.patch:1]: Does the LPA Questionnaire need sessions? I suspect it does, but if not then we don't need this patch

File: packages/lpa-questionnaire-web-app/src/config.js [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+        useNewUrlParser: true,
+        useUnifiedTopology: true,
+      },
+    },
+  },
+  documents: {
+    timeout: Number(process.env.DOCUMENTS_SERVICE_API_TIMEOUT || 10000),
+    url: process.env.DOCUMENTS_SERVICE_API_URL,
+  },
+  fileUpload: {
```
Comment (MrSimonEmms) [inline packages/lpa-questionnaire-web-app/src/config.js:26]: I suspect we won't need the `fileUpload` object (unless we're uploading files from this questionnaire)

File: packages/lpa-questionnaire-web-app/src/config.js [hunk 11]
```diff
@@ -0,0 +1,59 @@
...
+    },
+    tempFileDir: process.env.FILE_UPLOAD_TMP_PATH,
+    useTempFiles: process.env.FILE_UPLOAD_USE_TEMP_FILES === 'true',
+  },
+  logger: {
+    level: process.env.LOGGER_LEVEL || 'info',
+    redact: ['opts.body', 'config.db.session.uri', 'config.server.sessionSecret'],
+  },
+  server: {
+    limitedRouting: {
```
Comment (MrSimonEmms) [inline packages/lpa-questionnaire-web-app/src/config.js:41]: I suspect we won't need `limitedRouting` functionality

**Review by MrSimonEmms — CHANGES_REQUESTED** [review]
I suspect a lot of the actual app will change so haven't gone too far down there

File: packages/lpa-questionnaire-web-app/src/config.js (hunk 10 above)
Comment (davehlewis) [reply packages/lpa-questionnaire-web-app/src/config.js:26]: We will be uploading files for this, hence why I have left this in

File: packages/lpa-questionnaire-web-app/patches/connect-mongodb-session+2.4.1.patch (hunk 9 above)
Comment (davehlewis) [reply packages/lpa-questionnaire-web-app/patches/connect-mongodb-session+2.4.1.patch:1]: I believe it does, but can confirm that tomorrow

File: package-lock.json (hunk 8 above)
Comment (davehlewis) [reply package-lock.json:1655]: @shilpabasu22 can you confirm this?

Comment (shilpabasu22) [reply package-lock.json:1655]: I have added this under lpa-questionnaire-e2e-tests and not under main package.json file. Not sure why is it showing in that package-lock.json file. I can remove it or touch base with you tomorrow.

**Review by MrSimonEmms — APPROVED** [review]

**Pull request merged**